guestbook/views/delete_page.py

```diff
@@ -34,7 +36,3 @@ def delete_greeting():
 				using_task_queue.add_task_queue(greeting.author, greeting.content)
```

Comment:
xem lại chỗ đk này: if users.is_current_user_admin() or user and greeting.author == user:
using_task_queue.add_task_queue(greeting.author, greeting.content)
delete và sign gửi mail với nội dung giống nhau hay sao?